### PR TITLE
Working at Mattermost

### PR DESCRIPTION
**Pull request by it33**
- Drafting this to help document more about our culture

### CODE REVIEW

**Comment by yangchen1** [comment]
I really love this. +1

File: source/process/working-at-mattermost.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+7. **Focus on results:** Output matters, not hours worked. Enjoy life and don't burn out. 
+8. **Continual development:** Cross-training and rotation builds up everyone on the team. It's core to our culture. 
+9. **Global perspective:** Develop outstanding software in a multi-lingual, multi-cultural community. 
+10. **Friends around the world:** Build relationships with staff and contributors in dozens of countries. 
+
+----------------------------------------------------------
+Working Remotely 
+----------------------------------------------------------
+
+While we're headquartered in Palo Alto, California, like many open source companies, a large portion of Mattermost, Inc. works remotely from around the world.
```
Comment (lfbrock) [inline source/process/working-at-mattermost.rst]: Sentence is ambiguous (does "like many open source companies" apply to the Palo Alto or working remotely). 

Propose something like "We're headquartered in Palo Alto, California, but like many open source companies a large portion of Mattermost, Inc. works remotely from around the world."

File: source/process/working-at-mattermost.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+This gives us tremendous advantages: 
+
+1. **Access to top talent.** We can hire from anywhere.
+2. **More comfort.** People have 100% control of their surroundings. 
+3. **No commute.** Removing a 30-60 minute commute saves 250-500 hours a year. 
+4. **Time zone coverage.** It's easier to support customers any time of day, without being up at odd hours.
+5. **Results-focused culture.** Output matters more than hours at a desk. 
+
+Running a distributed, high performance organization
```
Comment (lfbrock) [inline source/process/working-at-mattermost.rst]: Need to complete this sentence

File: source/process/working-at-mattermost.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+The one requirement is having a strong internet connection for occassional video or audio conferences. Ethernet connections are preferred over Wifi, since Wifi can have interference. 
+
+2) Be explicit 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When you send a message, you take someone's full attention, so get across the relevant information as soon as you can.
+
+Don't say "Are you free to talk?", instead say "Catch up on ulimit issue Customer X is having?"
+
+Likewise, over communicate in writing when you can. The more people can read, the faster they can become effective.
```
Comment (lfbrock) [inline source/process/working-at-mattermost.rst]: "Over communicate" is confusing, is that what you meant? Generally over-communicating is referred to as a negative thing, so maybe phrase it differently (eg be thorough, or over communication is preferable to under communication) or add more context on what you mean here

File: source/process/working-at-mattermost.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+Find time after meetings and video conferences to get to know your team mates. Ask them about their weekends and how their life is outside of work.
+
+When you meet in person at off-sites and events, build on those relationships.  
+
+5) Be the change you want to see
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When something's not clearly documented, take the intiative to figure out what's going on and update the documentation for the people who come after you. 
+
+"Learn, Master, Teach" is a model we use to pass on training after you've been trained to help grow everyone at the company.
```
Comment (lfbrock) [inline source/process/working-at-mattermost.rst]: Maybe add a few more details on Learn Master Teach or link to other documentation to explain what it is

File: source/process/working-at-mattermost.rst (hunk 1 above)
Comment (it33) [inline source/process/working-at-mattermost.rst]: Thanks! Addressed: https://github.com/mattermost/docs/pull/519/commits/abc056217655aa6dad89400b176c4ba1f228a081

File: source/process/working-at-mattermost.rst (hunk 2 above)
Comment (it33) [inline source/process/working-at-mattermost.rst]: Thanks! Addressed: https://github.com/mattermost/docs/pull/519/commits/abc056217655aa6dad89400b176c4ba1f228a081

File: source/process/working-at-mattermost.rst (hunk 3 above)
Comment (it33) [inline source/process/working-at-mattermost.rst]: Thanks! Addressed: https://github.com/mattermost/docs/pull/519/commits/abc056217655aa6dad89400b176c4ba1f228a081

File: source/process/working-at-mattermost.rst (hunk 4 above)
Comment (it33) [inline source/process/working-at-mattermost.rst]: Thanks! Addressed: https://github.com/mattermost/docs/pull/519/commits/e53256052f598cbc9a95386318e06943a4f37938

**Comment by it33** [comment]
@lfbrock updated based on your feedback.

**Comment by lfbrock** [comment]
+1

**Pull request merged**